src/utils/data/columnsPerUserRole.ts

```diff
@@ -13,23 +13,15 @@ const COLUMNS_PER_ROLE: ColumnsPerRole = {
     APPLICATION_COLUMNS.STAGE,
     APPLICATION_COLUMNS.APPLICANT_ACTION,
   ],
-  reviewer1: [
+  reviewer: [
     APPLICATION_COLUMNS.SERIAL_NUMBER,
     APPLICATION_COLUMNS.LAST_ACTIVE_DATE,
     APPLICATION_COLUMNS.APPLICATION_NAME,
     APPLICATION_COLUMNS.ORGANISATION,
     APPLICATION_COLUMNS.STAGE,
     APPLICATION_COLUMNS.REVIEWER_ACTION,
   ],
-  reviewer2: [
-    APPLICATION_COLUMNS.SERIAL_NUMBER,
-    APPLICATION_COLUMNS.LAST_ACTIVE_DATE,
-    APPLICATION_COLUMNS.APPLICATION_NAME,
-    APPLICATION_COLUMNS.ORGANISATION,
-    APPLICATION_COLUMNS.STAGE,
-    APPLICATION_COLUMNS.REVIEWER_ACTION,
-  ],
-  supervisor: [
+  assigner: [
```

Comment:
Shouldn't "STAGE" still be visible for Assigner?

Reply:
And consolidator?

Reply:
Yes, good point. Adding that as well